### PR TITLE
Add Graffio workload and output bytes/s counter to benchmark 

### PR DESCRIPTION
**Pull request by igor-aptos**

### CODE REVIEW

File: crates/transaction-generator-lib/src/publishing/raw_module_data.rs [hunk 1]
```diff
@@ -490,19 +557,22 @@ pub static MODULES_FRAMEWORK_USECASES: Lazy<Vec<Vec<u8>>> = Lazy::new(|| { vec![
...
-		0, 0, 0, 0, 0, 2, 255, 141, 140, 177, 14, 130, 48, 16, 64, 247, 126, 69,
-		195, 194, 68, 193, 15, 112, 32, 70, 86, 23, 55, 194, 112, 180, 167, 65, 160, 215,
-		244, 26, 53, 49, 254, 187, 109, 26, 117, 37, 185, 229, 238, 222, 123, 189, 3, 61,
-		195, 21, 7, 97, 97, 69, 185, 151, 37, 172, 35, 48, 131, 33, 95, 138, 59, 122,
-		158, 200, 166, 243, 78, 53, 170, 41, 133, 232, 193, 24, 143, 204, 200, 131, 248, 163,
-		145, 40, 154, 231, 161, 237, 142, 69, 68, 12, 58, 180, 6, 173, 158, 18, 213, 186,
-		64, 220, 249, 152, 127, 144, 159, 35, 249, 146, 11, 105, 88, 146, 163, 84, 157, 231,
-		242, 253, 215, 144, 240, 234, 183, 23, 242, 157, 11, 103, 154, 209, 158, 198, 27, 234,
-		192, 27, 35, 33, 41, 21, 101, 39, 133, 62, 181, 7, 60, 183, 238, 0, 0, 0,
+		0, 0, 64, 68, 56, 48, 54, 49, 54, 65, 66, 70, 56, 48, 67, 49, 48, 51,
```
Comment (bchocho) [inline crates/transaction-generator-lib/src/publishing/raw_module_data.rs:560]: Any idea why this changes?

File: testsuite/module-publish/src/packages/complex/sources/smart_table_picture.move [hunk 2]
```diff
@@ -0,0 +1,85 @@
+// Copyright Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+module 0xABCD::smart_table_picture {
```
Comment (bchocho) [inline testsuite/module-publish/src/packages/complex/sources/smart_table_picture.move:4]: someone who knows move needs to review this

File: testsuite/single_node_performance.py [hunk 3]
```diff
@@ -334,6 +347,7 @@ def print_table(
                 "io g/s",
                 "exe g/s",
                 "g/t",
+                "out b/s"
```
Comment (bchocho) [inline testsuite/single_node_performance.py:350]: nit: b/s -> B/s

**Review by bchocho — APPROVED** [review]
LGTM, but someone who understands the graffio contract should review to see if this is expected/worst case behavior

**Review by sitalkedia — APPROVED** [review]
LG

File: testsuite/module-publish/src/packages/complex/sources/smart_table_picture.move (hunk 2 above)
Comment (msmouse) [reply testsuite/module-publish/src/packages/complex/sources/smart_table_picture.move:4]: I assume we copied from certain bad contract?

**Review by msmouse — APPROVED** [review]

File: testsuite/module-publish/src/packages/complex/sources/smart_table_picture.move (hunk 2 above)
Comment (igor-aptos) [reply testsuite/module-publish/src/packages/complex/sources/smart_table_picture.move:4]: yup 

https://github.com/aptos-labs/graffio/blob/main/move/sources/canvas/token.move#L223

File: crates/transaction-generator-lib/src/publishing/raw_module_data.rs (hunk 1 above)
Comment (igor-aptos) [reply crates/transaction-generator-lib/src/publishing/raw_module_data.rs:560]: either compiler changes, or someone made changes without compiling? but don't exactly know :)

File: execution/executor/src/components/chunk_output.rs [hunk 4]
```diff
@@ -261,6 +261,12 @@ pub fn update_counters_for_processed_chunk<T, O>(
     }
 
     for (txn, output) in transactions.iter().zip(transaction_outputs.iter()) {
+        if detailed_counters {
+            if let Ok(size) = bcs::serialized_size(output.get_transaction_output()) {
+                metrics::APTOS_PROCESSED_TXNS_OUTPUT_SIZE.inc_by(size as u64);
```
Comment (JoshLind) [inline execution/executor/src/components/chunk_output.rs:266]: Nice 😄

**Review by JoshLind — APPROVED** [review]
LGTM, thanks @igor-aptos! Also, I'd be interested to see the maximum number of changes we can push per transaction (before we hit the gas limits?). I assume `1K` is not the max? 🤔
